apps/sms/style/sms.css

```diff
@@ -37,6 +37,49 @@ gaia-header {
   z-index: 10;
 }
 
+/*
+  toast
+*/
+section[role="toast"] {
+  background: #4d4d4d;
+  overflow: hidden;
+  position: absolute;
+  z-index: 100;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  height: 6rem;
+  text-align: left;
+  opacity: .95;
+  margin: 0rem 1.5rem 5rem 1.5rem;
+}
+
+section[role="toast"] p {
+  font-size: 1.8rem;
+  font-weight: normal;
+  line-height: 2.2rem;
+  margin-left: 1.5rem;
+  margin-top: 2rem; 
+  padding: 0;
+  color: #ffffff;
+  text-align: left;
+  font-style: italic;
+  font-family: sans-serif;
+}
+
+section[role="toast"] p strong {
+  font-weight: 500;
+}
+
```

Comment:
I think we should have a separate CSS file toast.css or undo_toast.css if we decide to do our own css. But I think we should use the css for the existing role="notification" with specific classes to override what we need to override, it should make it easier to update it system-wide. For example the background color could be updated in one place.